src/styles/clock.scss

```diff
@@ -0,0 +1,68 @@
+@import "./variables.scss";
```

Comment:
Note from readme 
- Write all styles in `src/styles/main.scss`.

So maybe there is no need to use @import and we should write all our code in one file.